mkosi.finalize

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+[ "$1" == "final" ] || exit 0
+
+exit 0
+
+cd $OUTPUTDIR
+
+mount \
+ -o loop \
+ image.root.raw \
+ image.root.dir
```

Comment:
What happens if `image.root.dir` does not exist? If this directory created than?

Reply:
No, it is some left-over before I moved the squashfs file creation. completely to the Makefile.
I'll remove the file.